updateSub/index.js

```diff
@@ -0,0 +1,34 @@
+'use strict'
+
+
+const azureWrapper = require('../util/azureWrapper');
+const connect = require('../src/db');
+const verifyGithubAccessToken = require('../src/actions/verifyGithubAccessToken');
+
+module.exports = azureWrapper(async function (context, req) {
+    console.log('Hello There, I am a welcome message');
+    console.log('req', req);
+    const conn = await connect();
+    const Task = conn.model('Task');
+    const Subscriber = conn.model('Subscriber');
+
+    const task = await Task.create({
+        method: req.method,
+        url: req.url,
+        params: req.body
+    });
+    const params = { authorization: req.headers.authorization };
+
+    const res = await verifyGithubAccessToken({ task, conn })(params);
```

Comment:
Don't use `verifyGithubAccessToken()` here, it isn't really necessary. Just make sure:

1. The token exists
2. The token's `subscriberId` is equal to the provided subscriber's `_id`.